src/ecrc-frontend/src/components/page/consent/Consent.js

```diff
@@ -228,200 +222,41 @@ export default function Consent({
       eivPassDetailsResults: "eivPassDetailsResults"
     };
 
+    const singleCall = {
```

Comment:
nit: rename the var to indicate what the singleCall is? perhaps:
```
consolidatedCallPayload
```

Reply:
Changed